test/testSphereMani.jl

```diff
@@ -14,6 +14,7 @@ import Manifolds: identity_element
 #FIXME REMOVE! this is type piracy and not a good idea, for testing only!!!
 Manifolds.identity_element(::Sphere{2, ℝ}) = SVector(1.0, 0.0, 0.0)
 Manifolds.identity_element(::Sphere{2, ℝ}, p::AbstractVector) = SVector(1.0, 0.0, 0.0) # Float64[1,0,0]
+DFG.getPointIdentity(::Sphere{2, ℝ}) = SVector(1.0, 0.0, 0.0)
```

Comment:
perhaps just add a comment that this is an abuse of notation  (Sphere not being a group).  

I have been thinking about in future perhaps we should change this call to `getPointDefault` or `getPointFallback` and avoid use of identity on this context.  The reason for waiting on such a signature chance for me is first completing the solver upgrades to full Riemannian first and then come back.